pkg/test/echo/cmd/server/main.go

```diff
@@ -111,6 +119,7 @@ func init() {
 	rootCmd.PersistentFlags().IntSliceVar(&httpPorts, "port", []int{8080}, "HTTP/1.1 ports")
 	rootCmd.PersistentFlags().IntSliceVar(&grpcPorts, "grpc", []int{7070}, "GRPC ports")
 	rootCmd.PersistentFlags().IntSliceVar(&tcpPorts, "tcp", []int{9090}, "TCP ports")
+	rootCmd.PersistentFlags().IntSliceVar(&tlsPorts, "tls", []int{}, "Ports that are using TLS. These must be defined as http/grpc/tcp.")
```

Comment:
not sure I understand the second sentence of the description. You mean it's only supported for those protocols?

Reply:
I think the UX here isn't great. Basically if you want HTTPS and TCP over TLS you would pass `--http 80 --tcp 90 --tls=80,90`. The TLS flag is not like the other flags where its defining a new port, its modifying an existing port.

Improvements here are welcome but I couldn't come up with a great one, so I went with this